src/clp_logging/handlers.py

```diff
@@ -69,21 +70,23 @@ def setFormatter(self, fmt: Optional[logging.Formatter]) -> None:
 
             if found:
                 fmt._fmt = fmt_str.replace(found, "")
-                self.write(f" [WARN][clp_logging] replacing \'{found}\' with clp_logging timestamp")
+                self._warn(f"replacing '{found}' with clp_logging timestamp format")
             else:
                 fmt._fmt = DEFAULT_LOG_FORMAT
-                self.write(f" [WARN][clp_logging] replacing \'{fmt_str}\' with \'{DEFAULT_LOG_FORMAT}\'")
+                self._warn(f"replacing '{fmt_str}' with '{DEFAULT_LOG_FORMAT}'")
         else:
             fmt._fmt = " " + fmt_str
-            self.write(" [WARN][clp_logging] prepending clp_logging timestamp to formatter")
+            self._warn("prepending clp_logging timestamp to formatter")
 
         fmt._style = fmt._style.__class__(fmt._fmt)
         self.formatter = fmt
 
+    def _warn(self, msg: str) -> None:
+        self._write(" " + WARN_PREFIX + msg)
```

Comment:
Missing a space after the prefix, right? Using a `f"string"` might also make it more readable.

Reply:
Or maybe you meant for the `WARN_PREFIX` to contain a space after it?